pkgs/by-name/op/ophcrack/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, libsForQt5
+, fetchurl
+, autoreconfHook
+, pkg-config
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ophcrack";
+  version = "3.8.0";
+
+  src = fetchurl {
```

Comment:
Can you use (yacas-gui)[    description = "Easy to use, general purpose Computer Algebra System${lib.optionalString enableGui ", built with GUI."}";] and (yacas)[    description = "Easy to use, general purpose Computer Algebra System${lib.optionalString enableGui ", built with GUI."}";] to explain what I can do here?

Reply:
yes, that a great example.

you can add `enableGui` parameter to this derivation ( have it disabled by default )
then remove/rename ophcrack-cli to ophcrack-gui, and all it will do `ophcrack.override { enableGuui = true; }`

Reply:
im sure if i got this right, any correction here i will be happy to make changes.